app/controllers/api/v2/exchange_rates/base_controller.rb

```diff
@@ -2,6 +2,8 @@ module Api
   module V2
     module ExchangeRates
       class BaseController < ApiController
+        include NoCaching
```

Comment:
I guess this is easier than tracking the events that drive the exchange rate changes

Reply:
Yes, it is possible to generate appropriate ETags but its a bigger change since at the moment we lack an identifier for when the S3 contents are updated. 

Hence conclusion was to just remove the caching - that being said it sounds like the underlying issue wasn't ETags anyway but this is lowest hanging fruit